packages/pipeline/src/Pipeline.tsx

```diff
@@ -0,0 +1,285 @@
+import React, {
+  forwardRef,
+  createRef,
+  useState,
+  Ref,
+  ReactElement,
+  ReactNode,
+} from 'react';
+
+import PropTypes from 'prop-types';
+import { findLast } from 'lodash';
+
+import { uiColors } from '@leafygreen-ui/palette';
+import Tooltip from '@leafygreen-ui/tooltip';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { useElementNode, useMutationObserver } from '@leafygreen-ui/hooks';
+
+import Stage from './Stage';
+import Counter from './Counter';
+
+import {
+  getPipelineCounterTooltip,
+  isStageElement,
+  isElementOverflowed,
+} from './utils';
+
+import {
+  getRootStyle,
+  Size,
+  Variant,
+  layout,
+  colors,
+  pipelineAttr,
+  pipelineStages,
+  stageChevronAttr,
+} from './styles';
+
+export interface StateForStyles {
```

Comment:
Doesn't seem like we need to export this?

Reply:
Resolved by 9f54a5e